windows-puppet-agent/azuredeploy.parameters.json

```diff
@@ -12,25 +12,16 @@
       "value": "admin password"
     },
     "vmDnsName": {
-      "value": "unique dns name"
-    },
-    "imagePublisher": {
-      "value": "image Publisher"
-    },
-    "imageOffer": {
-      "value": "image Offer"
-    },
-    "imageSKU": {
```

Comment:
I'm just going to guess that `imageSKU` should not be a randomly generated value?